app/models/group.rb

```diff
@@ -100,6 +100,11 @@ def total_in_funded
       buckets.with_totals.where("status = 'funded'").sum("contrib.total")
   end
 
+  def total_in_unfunded
+      # amount of money in unfunded buckets
+      buckets.with_totals.where("status = 'live'").sum("contrib.total")
```

Comment:
@jlopker Some of the old buckets from the old state model might have `status = 'live'` and `archived_at = true`. This buckets are en reality cancelled and shouldn't be part of the `total_in_unfunded` value.

Reply:
@chime-mu ahh yes good catch! fixed!